web/src/engine/websites/ReadComicOnline.ts

```diff
@@ -13,19 +14,36 @@ function ChapterExtractor(anchor: HTMLAnchorElement) {
     };
 }
 
-function MangInfoExtractor(element: HTMLDivElement) { //needed because titles are truncated in link
-    const anchor = element.querySelector<HTMLAnchorElement>('a');
-    const dom = new DOMParser().parseFromString(element.getAttribute('title'), 'text/html');
-    return {
-        id: anchor.pathname,
-        title: dom.querySelector('p.title').textContent.trim()
-    };
-}
+const pageScript = `
```

Comment:
Why not using the script from [HakuNeko](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/KissComic.mjs#L55-L69)?

Reply:
Like i've explained in the PR message, with Hakuneko solution that a contribor did recently, we get a lot of blank.gif in haruneko. Dont ask me why 🤷 . The solution i've found was to increase delay. Like i've said, a fixed delay is not reliable.

I use setInterval to resolve the promise asap, but when "its done".

Reply:
I was referring to consider a CSS query instead of regex-ing the image array
e.g.,
```javascript
const links = [ ...document.querySelectorAll('#divImage img') ].map(img => img.src);
if(links.length > 0 && !links.some(link => /blank.gif/i.test(link))) {
    resolve(images);
}
```